library/src/main/java/com/wultra/android/mtokensdk/operation/OperationsService.kt

```diff
@@ -163,6 +170,34 @@ class OperationsService: IOperationsService {
         }
     }
 
+    private fun processServerTime(response: OperationListResponse, requestStarted: ZonedDateTime) {
+
+        val now = ZonedDateTime.now()
+        val requestDelayMilliseconds = now.toInstant().toEpochMilli() - requestStarted.toInstant().toEpochMilli()
+
+        // server does not support this feature
+        if (response.currentTimestamp == null) {
+            return
+        }
+
+        // Reject the value if the request took too long and we already have a server date.
+        // This is to avoid volatility of the value
+        if (serverDateShiftInMilliSeconds != null && requestDelayMilliseconds > 1_000) {
```

Comment:
It would be nice to extract the magic constant into the companion object and give it a nice name.